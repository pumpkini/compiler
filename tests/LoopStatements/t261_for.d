int main(){
     int i;
	int j;
     i = 0;
    for(; i < 3; i = i+1){
		for(j=i; j >= 0; j = j-1){
			Print(i, j);
		}
	}
}