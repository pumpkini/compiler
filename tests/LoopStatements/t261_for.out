00
11
10
22
21
20
